runner/metrics/metrics.go

```diff
@@ -0,0 +1,70 @@
+package metrics
+
+import (
+	"log/slog"
+	"time"
+
+	"github.com/cloudbase/garm/auth"
+	"github.com/cloudbase/garm/runner"
+)
+
+func CollectObjectMetric(r *runner.Runner, timer *time.Ticker) {
+
+	ctx := auth.GetAdminContext()
```

Comment:
This could accept a new parameter of type `context.Context` and return a new context with the appropriate admin fields set. If `nil` is passed, it can generate a `context.Background()` as it does now.

Reply:
done with https://github.com/cloudbase/garm/pull/216/commits/97f172eb513bda47d6658b81fd9bf262d687aa98